tests/assignment_four/test_arithmetic.py

```diff
@@ -0,0 +1,102 @@
+"""
+All unit tests which focus on parsing of operators.
+"""
+import pytest
+
+import calc
+
+
+def test_calc_can_parse_subtraction():
+    """
+    Test that ``Calc.parse()`` can correctly parse subtraction.
+    """
+    assert False  # Don't just make this True.
+
+
+def test_calc_can_parse_multiplication_by_one():
+    """
+    Test that ``Calc.parse()`` can correctly parse multiplication by one.
+    """
+    assert False  # Don't just make this True.
+
+
+def test_calc_can_parse_multiplication_of_positive_integers():
+    """
+    Test that ``Calc.parse()`` can correctly parse multiplication of two
+    positive integers.
+    """
+    assert False  # Don't just make this True.
+
+
+def test_calc_can_parse_addition_of_zero():
+    """
+    Test that ``Calc.parse()`` can correctly parse adding zero to an integer.
+    """
+    assert False  # Don't just make this True.
+
+
+def test_calc_can_parse_subtraction_of_zero():
+    """
+    Test that ``Calc.parse()`` can correctly parse subtracting zero from an
+    integer.
+    """
+    assert False  # Don't just make this True.
+
+
+def test_calc_can_parse_division_by_one():
+    """
+    Test that ``Calc.parse()`` can correctly parse a divisive identity.
+    """
+    assert False  # Don't just make this True.
+
+
+def test_calc_parses_integer_division_instead_of_floating_point_division():
+    """
+    Test that ``Calc.parse()`` does integer division rather than floating point
+    division. That is to say, parsing an expression that has a fractional
+    remainder should yield a truncated result.
+
+    Note:
+        Yes, this is an obscenely long function name.
```

Comment:
While I believe identifiers are unlimited in length 68 characters seems... excessive...